### DISCUSSION TITLE
Strict navigation options

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#305] [Q&A] [answered]
Function: Strict route following.
Suggestion **NEW**
- Keep: Non strict.
- Keep: Strict for both shaping & via points.
-  **Extra: Strict only for the via points.**

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the interesting idea!

With Cruiser this is very easy, I just need to switch a variable. :slightly_smiling_face: 

So we could have a 3-state setting for the strict route following:
- All points (current implementation)
- Via points
- Off

**Comment by devemux86** [comment]
<img src="https://github.com/devemux86/cruiser/assets/3484020/98935f94-64ea-4a27-afd5-707654bf02ec" width="50%"/>

**Comment by devemux86** [comment]
Available for testing in the latest:
- https://github.com/devemux86/cruiser/discussions/299

**Reply by Ciutnik** [reply]
A little note about the Polish translation. In this case, it should be "Off" = "Wyłączona" (with the letter "a" at the end, because in Polish it is feminine). The others are perfect! Thanks 👍

This time I waited for the implementation in the application, so I'm not wrong! 😉

**Reply by DolomitesTom** [reply]
If using the 2nd option, how many meters from the WP are considering the WP as taken?

When can the new things be tested via play store?

**Reply by devemux86** [reply]
@DolomitesTom it is similar to the current pass all (via + shaping) points, but only for via points.
A bit difficult to explain as it involves several navigation algorithms.

We will see about the next release, there are still a few things to improve.
But it's better to test with the beta versions. :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
I did and many things are different , for example Export settings is lost, only import is available in the application menu.

**Reply by devemux86** [reply]
@DolomitesTom 

The profiles (and their settings) are now handled in the new profile manager.
The settings have a new format. We keep for now the import of pref for compatibility.

Please see the announcement and instructions here:
- https://github.com/devemux86/cruiser/discussions/165#discussioncomment-6800089

And here, as this screen is still in progress:
- https://github.com/devemux86/cruiser/discussions/165#discussioncomment-6949064

**Comment by devemux86** [comment, marked as answer]
Strict navigation options are available in [Cruiser 3.0.32](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
